src/content-helper/dashboard-widget/top-posts/component-list-item.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TopPostData } from './model';
+import { impreciseNumber } from '../../../blocks/shared/functions';
+import PublishedLinkIcon from '../../../blocks/content-helper/icons/published-link-icon';
+import { getSmartShortDate } from '../../../blocks/shared/utils/date';
+
+interface TopPostListItemProps {
+	post: TopPostData;
+}
+
+/**
+ * Returns a single list item depicting a post.
+ *
+ * @param {TopPostData} post The Post to be shown.
+ */
+function TopPostListItem( { post }: TopPostListItemProps ): JSX.Element {
+	return (
+		<li className="parsely-top-post">
+			<div className="parsely-top-post-wrapper">
+
+				<div className="parsely-top-post-thumbnail">
+					<span className="screen-reader-text">Thumbnail</span>
+					<img src={ post.thumbUrlMedium } alt={ __( 'Post thumbnail', 'wp-parsely' ) } />
+				</div>
+
+				<div className="parsely-top-post-data">
+
+					<span className="parsely-top-post-views">
+						<span className="screen-reader-text">Number of Views</span>
+						{ impreciseNumber( post.views.toString() ) }
+					</span>
+
+					{ getPostTitleElement( { post } ) }
```

Comment:
This takes the user to edit view of the post ..... shouldn't it be in normal view mode?

Reply:
This is as per the requirements [here](https://github.com/Parsely/wp-parsely/issues/1221#issuecomment-1353319380). I've posted about this in internal communications, in case we decide to change it in the future.